### ISSUE TITLE
can´t import textures

### ISSUE THREAD

**Issue by santiagoolmos** [MrCrayfish/ModelCreator#49]
i have created some textures and when i try to import , the program says that the size is not correct, but their size is 16x16. i don´t know what is the problem. this is a example of image
![texturaestaciondesalud 2](https://cloud.githubusercontent.com/assets/13066474/8380361/99cb8c18-1c25-11e5-81e5-ffb1a427a1ba.png)

**Comment by mateuszdrwal** [comment]
are you sure you are using the latest commit? works for me
![works](https://cloud.githubusercontent.com/assets/11805733/8392050/ba1b0ba8-1cdc-11e5-89d8-62fb1cf47a68.png)
if you are using the pre release 4 use this:
https://www.dropbox.com/sh/njg2i8kczk4jmes/AADSYJ7RdAY7V8kcUULeJ90ma?dl=0

**Comment by santiagoolmos** [comment]
depending on which model I am using, I´m allowed to use it or not, for example, with the one that I´m working currently I can´t, but if I start a new one, I can, so I guess I will have to re do it. And yes,  I´m using the pre release- 4
![imagen1](https://cloud.githubusercontent.com/assets/13066474/8406823/aa240314-1e60-11e5-8172-166781b3ed4a.png) (this is the one I´m talking about)

**Comment by santiagoolmos** [comment]
also, thank you for the help :)

**Comment by mateuszdrwal** [comment]
yeah, that might just be an issue with the pre-release 4 so use the other one i linked and if it does not come up again close the issue :)

**Comment by santiagoolmos** [comment]
thank you very much!

**Comment by mateuszdrwal** [comment]
np! im happy to help :D

**Comment by micheel665** [comment]
issue still persists even with what mateuszdrwal linked

**Comment by micheel665** [comment]
I have come to the conclusion that the program doesnt like texture files with similar names?
I have a texture named sassacre1 and sassacre2 both 128x128. if I try and load them it gives me an error, but if I change sassacre1 to sas1 and then try and load it loads perfectly.
really odd stuff

**Comment by KnightMiner** [comment]
Actually, I noticed the same issue. It seems the program ignores numbers on names (the output also excludes the numbers in the name)

**Comment by MrCrayfish** [comment]
That sounds really strange, I will look into the issue.
